README.md

```diff
@@ -1,463 +1,161 @@
-[![BCH compliance](https://bettercodehub.com/edge/badge/kantord/LibreLingo?branch=main)](https://bettercodehub.com/)
-![David](https://img.shields.io/david/kantord/LibreLingo)
-
-<img align="right" src="/docs/branding/LibreLingo_Mascot/LightShadow/Turtle1_jetPack.svg" alt="" height="250" />
-
 <p align="center">
- <h1 align="center">LibreLingo</h1>
- <p align="right"><em>a community-owned language-learning platform</em></p>
- <h4 align="center">»<a href="https://librelingo.app/">&nbsp;Try&nbsp;LibreLingo&nbsp;now!&nbsp;</a>«</h4>
+  <img src="/docs/branding/LibreLingo_Mascot/LightShadow/Turtle1_jetPack.svg" height="250" />
+  <br/><br/>
+  <font size="6"><b>LibreLingo</b></font>
+  <br/>
+  <em>a community-owned language-learning platform</em>
+  <br/><br/>
+  <a href="https://librelingo.app/"><img src="https://img.shields.io/badge/Try_now-librelingo.app-darkcyan" /></a>
 </p>
+<hr/>
 
-<p align="center">
-  <a href="https://app.element.io/#/room/#space-librelingo:matrix.org">💬 Chat</a> •
-  <a href="#become-a-contributor">👩‍💻 Contribute!</a> •
-  <a href="https://github.com/sponsors/kantord">💵 Sponsor</a>  •
-  <a href="https://librelingo.app/docs/">📄 Development docs</a>  •
-  <a href="https://app.harmonizely.com/harmonizely-uoicqk">☎️ Schedule a call about Librelingo</a>
-</p>
+[![This project is using Percy.io for visual regression testing.](https://percy.io/static/images/percy-badge.svg)](https://percy.io/LibreLingo/LibreLingo)
+[![GitHub release (latest by date)](https://img.shields.io/github/v/release/kantord/LibreLingo?color=darkcyan)](https://github.com/kantord/LibreLingo/releases)
+![Build status](https://kantord.semaphoreci.com/badges/LibreLingo.svg?style=shields "Build status")
+[![ Sponsors](https://img.shields.io/github/sponsors/kantord?color=darkcyan)](https://github/sponsors/kantord)
+[![Matrix](https://img.shields.io/matrix/space-librelingo:matrix.org?color=%23000&label=chat%20on%20matrix&logo=matrix)](https://app.element.io/#/room/#space-librelingo:matrix.org)
+[![call](https://img.shields.io/badge/%E2%98%8E%EF%B8%8F-Schedule_a_call-darkcyan)](https://app.harmonizely.com/harmonizely-uoicqk)
+[![Mastodon](https://img.shields.io/mastodon/follow/000318010?domain=https%3A%2F%2Ffosstodon.org%2F&style=social)](https://fosstodon.org/@librelingo)
 
 
+## About
 
-## About LibreLingo
-
-LibreLingo's mission is to create a modern language-learning platform owned by the community of its users. All software is licensed under AGPLv3, which guarantees the freedom to run, study, share and modify the software. Course authors are encouraged to release their courses with free licenses.
-
- If you want to know why I built LibreLingo, [I recommend reading my article](https://dev.to/kantord/why-i-built-librelingo-280o). If you agree with LibreLingo's mission, [consider helping to build and maintain it](#become-a-contributor), and [please consider making a monetary contribution](https://github.com/sponsors/kantord).
-
-<details open="open">
```

Comment:
Why remove the table of contents? I think it was kinda useful as this readme is rather big

Reply:
certainly it is, and as I have mentioned in the pull request github already shows one on the top left corner of the readme, which sticks to the top of the screen making it accessiable anywhere of the readme but if we make one it goes beyond the screen, again you can change that

Reply:
oh, I had no idea about this feature. I'm not sure if it's well known or not. But I feel like it doesn't have great discoverability, basically to notice it, you need to already know that it exists :sweat_smile: 

I would say we need to cater to a more general audience. Some people who open the repo page might not be github users. I can imagine some of them might not have used github before :thinking: I haven't discovered this feature after several years of daily use of github :sweat_smile:

Reply:
added